scripts/socean_diet_data.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: latin-1 -*-
+#retriever
+from retriever.lib.templates import Script
+from retriever.lib.models import Table, Cleanup, correct_invalid_value
+from retriever import VERSION
+from pkg_resources import parse_version
+
+class main(Script):
+    def __init__(self, **kwargs):
+        Script.__init__(self, **kwargs)
+        self.title="A Southern Ocean dietary database"
+        self.citation="Ben Raymond, Michelle Marshall, Gabrielle Nevitt, Chris L. Gillies, John van den Hoff, Jonathan S. Stark, Marcel Losekoot, Eric J. Woehler, and Andrew J. Constable. 2011. A Southern Ocean dietary database. Ecology 92:1188."
+        self.name="socean-diet-data"
+        self.shortname="socean-diet-data"
+        self.ref="https://figshare.com/articles/Full_Archive/3551304"
+        self.description="Diet-related data from published and unpublished data sets and studies"
+        self.retriever_minimum_version='2.0.dev'
+        self.version='1.0.0'
+        self.urls={"zip": "https://ndownloader.figshare.com/files/5618823"}
+        self.cleanup_func_table = Cleanup(correct_invalid_value, missing_values=['', 'unknown'])
```

Comment:
since all the python files are following the check ` if parse_version(VERSION) <= parse_version("2.0.0"):` , lets use the same pattern. Do the same for `scripts/flensburg_food_web.py` above.
```
        self.title="A Southern Ocean dietary database"
        self.citation="Ben Raymond, Michelle Marshall, Gabrielle Nevitt, Chris L. Gillies, John van den Hoff, Jonathan S. Stark, Marcel Losekoot, Eric J. Woehler, and Andrew J. Constable. 2011. A Southern Ocean dietary database. Ecology 92:1188."
        self.name="socean-diet-data"
        self.ref="https://figshare.com/articles/Full_Archive/3551304"
        self.description="Diet-related data from published and unpublished data sets and studies"
        self.retriever_minimum_version='2.0.dev'
        self.version='1.0.0'
        self.urls={"zip": "https://ndownloader.figshare.com/files/5618823"}
        self.cleanup_func_table = Cleanup(correct_invalid_value, missing_values=['', 'unknown'])

        if parse_version(VERSION) <= parse_version("2.0.0"):
            self.shortname = self.name
            self.name = self.title
            self.tags = self.keywords
           self.cleanup_func_table = Cleanup(correct_invalid_value, nulls=['', 'unknown'])
```